server/init.lua

```diff
@@ -0,0 +1,26 @@
+-- Import section
+
+Event = require("event")
+
+local addressesConfigFile = "configuration.addresses"
+local addresses = require(addressesConfigFile)
+local addMachine = require("usecases.add-machine")
+local getMultiblockStatus = require("usecases.get-multiblock-status")
+
+--
+
+local function updateMachineList(_, address, _)
+    local comp = Component.proxy(address)
+    if comp.type == "waypoint" or comp.type == "gt_machine" or comp.type == "gt_batterybuffer" then
```

Comment:
Perhaps we should have these be constants or am enum, or maybe an array of things that we can do whatever lua's equivalent of Array.includes is. It'll make it easier to add more in the future if needed. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes